scripts/covid_tracking_project/historic_us_data/README.md

```diff
@@ -0,0 +1,17 @@
+# Importing The COVID Tracking Project's Historic US Data
+
+This directory contains artifacts required for importing
+The COVID Tracking Project's Historic US Data
+into Data Commons, along with scripts used to generate these artifacts.
+
+To generate `COVIDTracking_US.tmcf` and `COVIDTracking_US.csv`, run:
+
+```bash
+python3 preprocess_csv.py
+```
+
+#### Note:
+
+`COVIDTracking_US.mcf` is actually the same with `COVIDTracking_States.mcf`, so we don't need to actually check it.
```

Comment:
Do these files exist?

Reply:
Sorry for missing StatVars after the file name.